sdk/typespec/typespec_client_core/Cargo.toml

```diff
@@ -52,7 +52,7 @@ json = ["typespec/json"]
 reqwest = ["reqwest/native-tls"]
 reqwest_deflate = ["reqwest/deflate"]
 reqwest_gzip = ["reqwest/gzip"]
-reqwest_rustls = ["reqwest/rustls-tls-native-roots"]
+reqwest_rustls = ["reqwest/rustls-tls-native-roots-no-provider"]
```

Comment:
Lets put the comment you have in the CHANGELOG next to this line as well. I have a suspicion that down the road devs might wonder why we picked this feature flag. Your comment will remediate and avoid them pestering you ;)

Reply:
When the TOML is formatted, comments won't stay with the line. What we need to do - and I could start here with this create in this PR - is document the features. There's no built-in way with cargo (yet - there is a feature request or two) but it's common to put a Features section in the README. That would be in the same commit so people properly looking through commit details would see.

Reply:
Well, I suppose I could also put a comment to the right of the line as well. I'm going to keep it shorter.